recipes/chargemol/bld.bat

```diff
@@ -0,0 +1,11 @@
+REM use pre-compiled executable for windows
+mkdir %PREFIX%\bin
+
+COPY chargemol_FORTRAN_09_26_2017\compiled_binaries\windows\Chargemol_09_26_2017_windows_64bits_parallel_command_line.exe %PREFIX%\bin\chargemol.exe
```

Comment:
We are OK with repackages but we do have some success with Fortran on Windows. conda-forge has flang and we can probably try msys2 gfortran too. Do you want to try those first?

Also, when shipping binaries, be sure you have all licenses for everything it links on the final package. (I'm guessing this is kind of "standalone," right? If so you should be good here.)

Reply:
thanks for the quick reply!

> We are OK with repackages but we do have some success with Fortran on Windows. conda-forge has flang and we can probably try msys2 gfortran too. Do you want to try those first?

I can give this a try. Do you happen to have an example in mind of a package that uses fortran + openmp on windows?

> Also, when shipping binaries, be sure you have all licenses for everything it links on the final package. (I'm guessing this is kind of "standalone," right? If so you should be good here.)

I think so

Reply:
@ocefpaf 
I'm just asking since this package needs openmp support and [the docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp) only mention MacOS and Linux (happy to make a PR if there is support for OpenMP on Windows as well).

Reply:
I'm not familiar with Windows to answer that. Maybe @isuruf knows the right way to do openmp on Windows.

Reply:
Yes, `llvm-openmp` is supported on windows. You'll have to use `clang-cl.exe` or `clang.exe` to use that.

Reply:
Thanks! This is a fortran code, so I guess I need to use `flang.exe`.

I'm trying but [I'm getting](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=266398&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=666)
```
(%BUILD_PREFIX%) %SRC_DIR%\chargemol_FORTRAN_09_26_2017\sourcecode_windows_command_line>flang.exe -fopenmp -o Chargemol_09_26_2017_windows_parallel   module_precision.f08   module_global_parameters.f08   module_common_variable_declarations.f08   module_quote.f08   module_string_utilities.f08   module_matrix_operations.f08   module_reshaping_functions.f08   module_CM5_parameters.f08   module_calculate_atomic_polarizabilities_upper_bound.f08   module_read_job_control.f08   module_read_wfx.f08   module_gaussian_functions.f08   module_compute_CM5.f08   module_check_atomic_reference_polarizabilities_available.f08   module_charge_center_positions_and_parallelpiped.f08   module_generate_kApoints_kBpoints_kCpoints.f08   module_check_grid_spacing.f08   module_add_missing_core_density.f08   module_oxidation_density.f08   module_initialize_atomic_densities.f08   module_align_periodic_atoms.f08   module_read_spin_density_cube_files.f08   module_format_valence_and_total_cube_densities.f08   module_format_valence_cube_density.f08   module_format_total_cube_density.f08   module_format_xsf_densities.f08   module_check_noncollinear_XC_functional.f08   module_gen_dens_grids_from_gaussian_basis_set_coeff.f08   module_atomic_symbol_to_number.f08   module_compute_dominant_atom_volumes.f08   module_format_vasp_densities.f08   module_run_valence_core_densities.f08   module_core_iterator.f08   module_local_multipole_moment_analysis.f08   module_update_atomic_densities.f08   module_DDEC3_valence_iterator.f08   module_DDEC6_valence_iterator.f08   module_compute_center_of_mass.f08   module_total_multipole_moment_analysis.f08   module_cloud_penetration.f08   module_atomic_number_to_symbol.f08   module_generate_atomic_polarizability_upper_bound_file.f08   module_generate_net_atomic_charge_file.f08   module_spin_functions.f08   module_generate_atomic_spin_moment_file.f08   module_collinear_spin_moments_iterator.f08   module_noncollinear_spin_moments_iterator.f08   module_generate_bond_order_file.f08   module_determine_whether_pair_overlap_is_significant.f08   module_calculate_final_BOs.f08   module_integrate_bonding_terms.f08   module_initialize_bond_pair_matrix.f08   module_compute_local_atomic_exchange_vectors.f08   module_prepare_BO_density_grids.f08   module_perform_bond_order_analysis.f08   module_compute_atomic_radial_moments.f08   module_print_overlap_populations.f08   module_print_atomic_densities_file.f08   chargemol.f08 
flang.exe: error: Fortran frontend to LLVM command failed due to signal (use -v to see invocation)
clang version 5.0.0 
Target: x86_64-pc-windows-msvc
Thread model: posix
InstalledDir: C:\Miniconda\conda-bld\chargemol_1611098245203\_build_env\Library\bin
flang.exe: note: diagnostic msg: PLEASE submit a bug report to  and include the crash backtrace, preprocessed source, and associated run script.
flang.exe: note: diagnostic msg: Error generating preprocessed source(s) - no preprocessable inputs.
```

Do you spot any obvious issues in the invocation or does this look like a compiler problem?

Reply:
Yeah, that's a bug. Maybe we can try gfortran from msys2. `compiler('m2w64_fortran')`

Reply:
Thanks! 

I'm trying it [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=266670&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=628). Essentially I'm now running
```
gfortran.exe -fopenmp -o Chargemol_09_26_2017_windows_parallel   module_precision.f08   ...
```
But on the next line the `Chargemol_09_26_2017_windows_parallel` I'm trying to build cannot be found.

Does the command seem right? I'm a bit puzzled that there is no output whatsoever from the `gfortran.exe` line, not even an error message.

Sorry if there is some obvious thing I'm missing here, I'm just completely new to building on windows for conda-forge.

Reply:
Ok, it seems on windows gfortran adds `.exe` to the specified executable name.